src/Sulu/Component/Websocket/MessageDispatcher/MessageHandlerContext.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Component\Websocket\MessageDispatcher;
+
+use Sulu\Component\Websocket\ConnectionContext\ConnectionContextInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+/**
+ * Message handler connection context
```

Comment:
Maybe you can add a bit more information here? What kind of values does this object hold?